pkg/operator/rollout/rollout.go

```diff
@@ -73,6 +77,15 @@ func Released(obj metav1.Object) bool {
 	return present
 }
 
+func Cascaded(obj metav1.Object) bool {
```

Comment:
Maybe this should be `Cascading`? The other annotations are telling us "this thing happened in the past" -- a change _was scheduled_ or _was released_. This annotation is more of an on/off switch -- "is this parent currently cascading or not?"

Also add a doc comment.

Reply:
Yeah that makes sense, done and done.